src/components/AddressForm.tsx

```diff
@@ -126,7 +126,7 @@ function AddressForm({
                     }
                 }
             } else if (!CONST.GENERIC_ZIP_CODE_REGEX.test(values?.zipPostCode?.trim()?.toUpperCase() ?? '')) {
-                errors.zipPostCode = translate('privatePersonalDetails.error.incorrectZipFormat');
+                errors.zipPostCode = translate('privatePersonalDetails.error.incorrectZipFormat', undefined);
```

Comment:
This looks odd, why do you have to add undefied here? `zipFormat` is an optional argument:

```ts
zipFormat?: string
```